LICENSE

```diff
@@ -1,674 +0,0 @@
-                    GNU GENERAL PUBLIC LICENSE
```

Comment:
This is critical! We need to have GPL v3 license in our repo

Reply:
Gonna add it now